cms/utils/setup.py

```diff
@@ -33,6 +35,14 @@ def validate_settings():
         raise ImproperlyConfigured("django CMS requires django.template.context_processors.request in "
                                    "'django.template.backends.django.DjangoTemplates' context processors.")
 
+    if (
+        hasattr(settings, "SEND_BROKEN_LINK_EMAILS") and  # noqa: W504
+        "django.middleware.common.BrokenLinkEmailsMiddleware" not in getattr(settings, "MIDDLEWARE", [])
+    ):
+        warnings.warn('The setting "SEND_BROKEN_LINK_EMAILS" will not be honored by django CMS as of version 4.1. '
```

Comment:
Similar to the discussion here, let's add `stacklevel=2` to the `warnings.warn` method, to make it easier for the end users of this to spot where they are using this in their code.

https://github.com/django-cms/django-cms/pull/7410#discussion_r1006064988

Reply:
Hi @vinitkumar ! Thanks, but the `SEND_BROKEN_LINK_EMAIL` is a setting which always resides in the settings file and I will not find on the stack. Hence, for this case, the stack level does not help. 

But the `stacklevel=2` is great stuff. Thank you so much. Before reading your other comment I was not aware of it.